src/app/dataset-view/additional-components/overview-component/overview.component.html

```diff
@@ -49,13 +49,13 @@ <h2 class="box-title align-items-center pt-3">Keywords</h2>
                     </p>
                 </div>
             </ng-template>
-            <div class="my-4" *ngIf="canAddSetPollingSource" data-test-id="add-set-polling-source-block">
+            <div class="my-4" data-test-id="add-set-polling-source-block">
                 <p>
                     You can
                     <a class="text-decoration-none" (click)="navigateToAddPollingSource()"> Add data source</a>
                 </p>
             </div>
-            <div class="my-4" *ngIf="canAddSetTransform" data-test-id="add-set-transform-block">
+            <div class="my-4" data-test-id="add-set-transform-block">
```

Comment:
Same here

Reply:
It was for testing, forgot to revert, and already moved it back